.ci/pytorch/common_utils.sh

```diff
@@ -135,10 +135,16 @@ function install_filelock() {
 
 function install_triton() {
   local commit
+  commit=$(get_pinned_commit triton)
+  local short_hash
+  short_hash=$(echo "${commit}"|cut -c -10)
+  local index_url
+  index_url=https://donwload.pytorch.org/whl/nightly/cpu
   if [[ "${TEST_CONFIG}" == *rocm* ]]; then
     echo "skipping triton due to rocm"
+  elif pip install --no-deps "pytorch-triton==2.0.0+${short_hash}" --index-url "${index_url}"; then
+     echo "Using prebuilt version"
```

Comment:
Nit: May be print the hash here to help debugging (instead of clicking around for the pinned commit)